### PR TITLE
goreleaser updates

### PR DESCRIPTION
**Pull request by redhatrises**
- Update goreleaser
- Add usage template
- Allow Makefile and goreleaser to build version into releases

### CODE REVIEW

**Comment by redhatrises** [comment]
This should also fix #170

File: .goreleaser.yml [hunk 1]
```diff
@@ -22,10 +33,10 @@ nfpm:
...
 brew:
-  name: oscalkit
+  name: compliance-masonry
   github:
     owner: opencontrol
-    name: homebrew-oscalkit
+    name: homebrew-compliance-masonry
   commit_author:
     name: OpenControl
     email: dev@open-control.org
```
Comment (shawndwells) [inline .goreleaser.yml]: Does this EMail exist?

File: cmd/masonry/masonry_test.go [hunk 2]
```diff
@@ -12,19 +12,24 @@ import (
...
+Commands:
   diff        Compliance Diff Gap Analysis
   docs        Create compliance documentation
   export      Export to consolidated output
   get         Install compliance dependencies
   help        Help about any command
+  version     Display version
 
-Flags:
+Options:
```
Comment (shawndwells) [inline cmd/masonry/masonry_test.go]: given the ``usage`` output, should this be renamed to ``command-options``?

File: cmd/masonry/masonry_test.go [hunk 3]
```diff
@@ -12,19 +12,24 @@ import (
 
 var usage = `
 Usage:
-  masonry [command]
+  masonry [global-options] COMMAND [command-options]
```
Comment (shawndwells) [inline cmd/masonry/masonry_test.go]: where are the ``global-options`` documented?

Comment (redhatrises) [reply cmd/masonry/masonry_test.go]: @shawndwells In the help/usage outputs for each of the commands. For example, `masonry help diff`

File: cmd/masonry/masonry_test.go (hunk 2 above)
Comment (redhatrises) [reply cmd/masonry/masonry_test.go]: No. It shouldn't.

File: .goreleaser.yml (hunk 1 above)
Comment (redhatrises) [reply .goreleaser.yml]: @shawndwells figured you would know more than I about that.

File: cmd/masonry/masonry_test.go (hunk 2 above)
Comment (shawndwells) [reply cmd/masonry/masonry_test.go]: Bah

Comment (shawndwells) [reply cmd/masonry/masonry_test.go]: I mean you're right.

default ``help`` behavior vs ``help get``

File: .goreleaser.yml (hunk 1 above)
Comment (shawndwells) [reply .goreleaser.yml]: Not something that's being changed in the PR, so no reason to hold a merge, just caught my eye.

**Pull request merged**